doc/reference.bib

```diff
@@ -7,6 +7,35 @@ @Manual{R
     url = {https://www.R-project.org/},
   }
 
+@article{scikit-learn,
```

Comment:
I notice that many things are tagged "@article", I expected this to be @software (though I'm very new to this .bib idea so I might be wrong). What do you think?

Additionally, these things also need to be referenced in the RMD file itself - not just in the .bib file. You can do this by adding `[@label]` tags into the RMD file (e.g. `[@scikit-learn]` in this case).

Reply:
about the @software, I'll check that. I'm not sure as well, I just saw some examples in google and copied that haha.

Reply:
I've added this in the RMD file. I'm not sure why you cannot see it. take a look at my file:

Reply:
![image](https://user-images.githubusercontent.com/94653774/143661037-9613f8b7-7d8f-4125-b6ad-c72bfa9da3db.png)

Reply:
I tried to update everything again.

Reply:
Oh yes, you're absolutely right, I think I didn't refresh my browser or something.